### DISCUSSION TITLE
Stacked Bar/ Column Chart by Species Over Time?

### DISCUSSION THREAD

**Discussion by AlaskanAstro** [Nachtzuster/BirdNET-Pi#76] [Ideas]
Does anyone know if it's possible to export the data from birdnet and create a stacked column chart? Something visually similar to this: 
![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/112118866/8901bae7-05f5-4d6a-acfc-8f93e2634d89)

I think it would be a very neat way to display detected birds over the year.

**Comment by lloydbayley** [comment]
Well, Grafana has a SQLite plugin but I've not tested it with BirdNet. It might be worth a look!

**Reply by AlaskanAstro** [reply]
Unfortunately my own skillset pretty much hits its limits with following a tutorial to do the 2 commands over SSH to install BirdNet. :) But it would be really neat is someone else explored this route.

**Comment by Nachtzuster** [comment]
@AlaskanAstro I see you aked the question here too - I missed that :smile: Repeating:

The data is all saved in a SQLite db: /home/<user>/BirdNET-Pi/scripts/birds.db the ui uses this one.
There is also a 'raw' csv file: /home/<bird>/BirdNET-Pi/BirdDB.txt

the difference is that the SQLite file is updated if you delete a detection in the UI, while the raw csv file is not

You can grab the files with Tools>File Manager
